src/PrestaShopBundle/Form/Admin/Sell/Product/Options/ProductAttachmentsType.php

```diff
@@ -0,0 +1,112 @@
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+class ProductAttachmentsType extends TranslatorAwareType
+{
+    /**
+     * @var UrlGeneratorInterface
+     */
+    private $urlGenerator;
+
+    public function __construct(
+        TranslatorInterface $translator,
+        array $locales,
+        UrlGeneratorInterface $urlGenerator
+    ) {
+        parent::__construct($translator, $locales);
+        $this->urlGenerator = $urlGenerator;
+    }
+
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('attached_files', EntitySearchInputType::class, [
+                'entry_type' => AttachedFileType::class,
+                'layout' => EntitySearchInputType::TABLE_LAYOUT,
+                'required' => false,
+                'label' => false,
+                'remote_url' => $this->urlGenerator->generate('admin_attachments_search', ['searchPhrase' => '__QUERY__']),
+                'placeholder' => $this->trans('Search file', 'Admin.Catalog.Feature'),
+                'empty_state' => $this->trans('No files associated', 'Admin.Catalog.Feature'),
+            ])
+            ->add('add_attachment_btn', IconButtonType::class, [
+                'label' => $this->trans('Add new file', 'Admin.Catalog.Feature'),
+                'icon' => 'add_circle',
+                'type' => 'link',
+                'attr' => [
+                    'data-success-create-message' => sprintf('%s %s',
+                        $this->trans(
+                            'Attachment was successfully created and added to the selection.',
+                            'Admin.Catalog.Feature'
+                        ),
+                        $this->trans(
+                            'This window will close automatically.',
+                            'Admin.Global'
+                        )
+                    ),
+                    'class' => 'btn-outline-secondary add-attachment',
+                    'href' => $this->urlGenerator->generate('admin_attachments_create', [
+                        'liteDisplaying' => true,
+                        'saveAndStay' => true,
+                    ]),
+                ],
+            ])
+        ;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        parent::configureOptions($resolver);
+        $resolver->setDefaults([
+            'label' => $this->trans('Attached files', 'Admin.Catalog.Feature'),
+            'label_tag_name' => 'h2',
+            'label_subtitle' => $this->trans('Select the files (instructions, documentation, recipes, etc.) your customers can directly download on this product page.', 'Admin.Catalog.Feature') .
+                '<br>' .
+                $this->trans(
+                    'Need to browse all files? Go to [1]Catalog > Files[/1]',
```

Comment:
I suggest replacing this sentence with a simple link, like: '[1]Browse all files[/1]

Reply:
I'm not sure it's exactly the same use case, if you are referring to the wording in the Basic tab under the shortcuts
Because they are links to the tab mapping the shortcuts fields, they are internal links inside the same page to access more fields and more settings (hence the "Advanced settings")

But this one is an external link to another page that opens a new tab, so it's not an "Advanced settings" link

Reply:
As discussed on Slack, here's what we can do :-) 
![image](https://user-images.githubusercontent.com/70583503/127347676-ca85932a-9158-4d89-9ca0-c524b5e077ab.png)